### PR TITLE
Add a note saying that artifact store information must be saved individually

### PR DESCRIPTION
**Pull request by nabenabe0928**
## Motivation

As artifact store information for each artifact store is not provided, I added a note to warn this. 

## Description of the changes

- Add a note in the doc-string

### CODE REVIEW

**Comment by y0z** [comment]
@c-bata, @porink0424 Could you review this PR?

**Review by c-bata — CHANGES_REQUESTED** [review]
@nabenabe0928 The change seems not to be reflected on the documentation. Could you check the following page?
https://optuna--5530.org.readthedocs.build/en/5530/reference/artifacts.html#optuna.artifacts.download_artifact

File: optuna/artifacts/_download.py [hunk 1]
```diff
@@ -18,6 +18,11 @@ def download_artifact(artifact_store: ArtifactStore, artifact_id: str, file_path
             The identifier of the artifact to download.
         file_path:
             A path to save the downloaded artifact.
+
+    .. note:
```
Comment (nabenabe0928) [inline optuna/artifacts/_download.py:22]:
```suggestion
    .. note::
```

**Review by nabenabe0928 — COMMENTED** [review]
I found an error based on the notice by c-bata.

**Comment by nabenabe0928** [comment]
@c-bata 
The problem has been fixed:)
image

File: optuna/artifacts/_download.py [hunk 2]
```diff
@@ -18,6 +18,11 @@ def download_artifact(artifact_store: ArtifactStore, artifact_id: str, file_path
             The identifier of the artifact to download.
         file_path:
             A path to save the downloaded artifact.
+
+    .. note::
+        Optuna does not provide any API to save artifact store information linked to each artifact,
```
Comment (porink0424) [inline optuna/artifacts/_download.py:23]: The phrase `artifact store information` might be a bit ambiguous. How about providing a specific example to clarify?

**Review by c-bata — COMMENTED** [review]
Thank you for the update. I should have pointed this out at the beginning, but I don't quite understand the motivation for this change. Generally, information about the ArtifactStore is specified in the source code and I don't see the necessity to store it separately. There are similar issues with the way Optuna Storage is specified, but I think it's obvious, so that it's not specified in the documentation.

What do you think?

**Comment by nabenabe0928** [comment]
@c-bata 
You are right, the artifact store information is obvious for users, so I will close this PR.

**Pull request closed without merging**